lib/config/migrations/custom/base-branches-migration.ts

```diff
@@ -0,0 +1,14 @@
+import is from '@sindresorhus/is';
+import { AbstractMigration } from '../base/abstract-migration';
+
+export class BaseBranchesMigration extends AbstractMigration {
+  override readonly deprecated = true;
+  override readonly propertyName = 'baseBranches';
```

Comment:
you should use the rename migration instead

Reply:
Yes. 

I wasn't able to handle the case where baseBranch and baseBranches both were present in the config. 

I will let that case be unhandled as it will be bad config practice anyway.